src/sqlfluff/dialects/dialect_redshift.py

```diff
@@ -585,20 +585,25 @@ class TableConstraintSegment(BaseSegment):
 
     type = "table_constraint"
 
```

Comment:
Suggest you review this next section with whitespace off:

<img width="270" alt="image" src="https://user-images.githubusercontent.com/10931297/169896229-9e47701c-e9d2-4b14-8f46-0f3cddef7dda.png">

Reply:
Ignore. Changed this quite a bit since then.